packages/platform/platform-multer/src/common/middlewares/PlatformMulterMiddleware.ts

```
@@ -0,0 +1,46 @@
+        fields: Field[];
+        options: Options;
+      }>(PLATFORM_MULTER_OPTIONS);
+
+      const settings: Options = {
+        ...constant<Options>("multer", {}),
+        ...options
+      };
+
+      /* istanbul ignore next */
+      if (settings.storage) {
+        delete settings.dest;
```

Biome (1.9.4)

[error] Avoid the delete operator which can impact performance.

Unsafe fix: Use an undefined assignment instead.


(lint/performance/noDelete)